test/api_test.rb

```diff
@@ -249,11 +247,22 @@ class WorkerWithTags
       ActiveJob::Base.logger = nil
     end
 
+    describe "delayed extensions" do
```

Comment:
new testing scope should make finding the gem-specific tests easier to find.

The rest of the tests are copied from sidekiq and can probably be removed. This gem was created with original intention to be quick and dirty good enough, with eventually being honed.